shared/src/commonMain/kotlin/com/fbada006/shared/utils/ImagePicker.kt

```diff
@@ -0,0 +1,13 @@
+package com.fbada006.shared.utils
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.ImageBitmap
+
+expect class ImagePicker {
```

Comment:
Why do we have two classes? Factory and the Picker. What is the distinction?

Reply:
The factory is responsible for creating platform specific instances of the ImagePicker. 
ImagePicker handles the individual selection process for Android and Ios